tests/test_node_provider_db.py

```diff
@@ -139,6 +139,14 @@ def test_channel_lookup_crud():
     assert ('test-dummy-principal-2', 'dummy-slack-channel-2', 'dummy-telegram-chat-2', 'dummy-telegram-channel-2') in channels
     assert ('test-dummy-principal-1', 'dummy-slack-channel-3', 'dummy-telegram-chat-3', 'dummy-telegram-channel-3') in channels
 
+    channels = node_provider_db.get_channels_as_dict()
+    assert channels['test-dummy-principal-1'] == \
+        {'node_provider_id': 'test-dummy-principal-1', 'slack_channel_name': 'dummy-slack-channel-3', 
+         'telegram_chat_id': 'dummy-telegram-chat-3', 'telegram_channel_id': 'dummy-telegram-channel-3'}
```

Comment:
Just to make sure we are on the same page. This means that if there are any duplicate entries in the channel_lookup table, the channel that will be selected will always be the latest entry correct?